examples/trivial-hetzner-cloud.nix

```diff
@@ -0,0 +1,12 @@
+{
+  resources.sshKeyPairs.ssh-key = {};
+
+  machine = { config, pkgs, ... }: {
+    services.openssh.enable = true;
+
+    deployment.targetEnv = "hetznerCloud";
+    deployment.hetznerCloud.serverType = "cx11";
+
+    networking.firewall.allowedTCPPorts = [ 22 ];
```

Comment:
Same here, as it's [done by the `sshd` module](https://github.com/NixOS/nixpkgs/blob/16b3217148b02ad5a2149b500dbfcfa08d7b46d7/nixos/modules/services/networking/ssh/sshd.nix#L410) (`openFirewall` is [enabled by default](https://github.com/NixOS/nixpkgs/blob/16b3217148b02ad5a2149b500dbfcfa08d7b46d7/nixos/modules/services/networking/ssh/sshd.nix#L142)).